tests/test_flobject.py

```diff
@@ -408,7 +408,6 @@ def test_primitives():
     assert r.g_1.b_3() is False
     r.g_1.s_4 = "foo"
     assert r.g_1.s_4() == "foo"
-    return True
```

Comment:
https://docs.pytest.org/en/7.2.x/deprecations.html#returning-non-none-value-in-test-functions